SuperBuild/External_CCPi-Regularisation-Toolkit.cmake

```
@@ -81,6 +81,7 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
       # external projects
       CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env CIL_VERSION=${${proj}_TAG} ${CMAKE_COMMAND} ${${proj}_SOURCE_DIR}
         -DCMAKE_INSTALL_PREFIX:PATH=${${proj}_INSTALL_DIR}
+        -DCMAKE_BUILD_TYPE:STRING=Release
```

I'm guessing this should be
```suggestion
        -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
```
@rijobro, what do you think?

---

That's interesting, yesterday evening I didn't see this comment, but well I wasn't paying too much attention.